packages/language-server/src/api.ts

```diff
@@ -19,3 +20,25 @@ export const SERVER_PATH: string = isDevelopmentRun
   ? /* istanbul ignore else - no tests (yet?) on bundled artifacts */
     sourcesPath
   : bundledPath;
+
+export async function getNodeName(
+  text: string,
+  offsetAt: number,
+  cachePath?: string | undefined,
+  framework?: string | undefined,
+  ui5Version?: string | undefined
+): Promise<string | undefined> {
+  const ui5Node = await getUI5NodeName(
+    offsetAt,
+    text,
+    undefined,
+    cachePath,
+    framework,
+    ui5Version
+  );
+  return ui5Node
```

Comment:
I think too much is done here in a single expression.
Can you split this up into multiple statements for readability?